package.json

```diff
@@ -1,111 +1,125 @@
-        "react-csv": "^2.2.2",
-        "react-day-picker": "^8.10.1",
-        "react-dom": "^18.3.1",
-        "react-dropzone": "^14.2.3",
-        "react-helmet-async": "^2.0.5",
-        "react-icons": "^5.2.1",
-        "react-infinite-scroll-component": "^6.1.0",
-        "react-markdown": "^9.0.1",
-        "react-quill": "^2.0.0",
-        "react-redux": "^9.1.2",
-        "react-router": "^6.23.1",
-        "react-router-dom": "^6.23.1",
-        "react-scroll": "^1.9.0",
-        "react-spinners": "^0.13.8",
-        "react-spinners-kit": "^1.9.1",
-        "react-syntax-highlighter": "^15.5.0",
-        "react-toastify": "^10.0.5",
-        "react-transition-group": "^4.4.5",
-        "rehype-sanitize": "^6.0.0",
-        "remark-gfm": "^4.0.0",
-        "sass": "^1.77.1",
-        "styled-components": "^6.1.11",
-        "validator": "^13.12.0"
-    },
-    "devDependencies": {
-        "@types/react": "^18.3.2",
-        "@types/react-dom": "^18.3.0",
-        "@vitejs/plugin-react": "^4.2.1",
-        "autoprefixer": "^10.4.19",
-        "eslint": "^8.0.0",
-        "eslint-config-prettier": "^9.1.0",
-        "eslint-config-standard": "^17.1.0",
-        "eslint-plugin-import": "^2.29.1",
-        "eslint-plugin-n": "^16.0.0",
-        "eslint-plugin-no-relative-import-paths": "^1.5.4",
-        "eslint-plugin-promise": "^6.1.1",
-        "eslint-plugin-react": "^7.34.1",
-        "gh-pages": "^6.1.1",
-        "husky": "^9.0.11",
-        "lint-staged": "^15.2.2",
-        "postcss": "^8.4.38",
-        "postcss-styled-syntax": "^0.6.4",
-        "prettier": "^3.2.5",
-        "stylelint": "^16.5.0",
-        "stylelint-config-standard": "^36.0.0",
-        "stylelint-plugin-styled-components": "^1.0.3",
-        "tailwindcss": "^3.4.3",
-        "vite": "^5.2.11"
-    },
-    "husky": {
-        "hooks": {
-            "pre-commit": "lint-staged"
-        }
-    },
-    "lint-staged": {
-        "*.{js,jsx}": [
-            "eslint --cache --fix",
-            "stylelint --fix"
-        ],
-        "*.{js,jsx,css,md}": "prettier --write"
-    },
-    "eslintConfig": {
-        "settings": {
-            "react": {
-                "version": "detect"
-            }
-        }
+  "homepage": "https://thecyberhub.org/",
+  "name": "thecyberhub.org",
+  "private": true,
+  "version": "0.53.14",
+  "type": "module",
+  "scripts": {
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d dist",
+    "dev": "vite",
+    "build": "NODE_OPTIONS=\"--max-old-space-size=4096\" vite build",
+    "preview": "vite preview",
+    "prepare": "husky install",
+    "lint:fix": "eslint --fix --ext .js,.jsx .",
+    "lint:css": "stylelint **/*.jsx",
+    "lint:css-fix": "stylelint **/*.jsx --fix",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build"
+  },
+  "dependencies": {
+    "@fortawesome/free-solid-svg-icons": "^6.5.2",
+    "@fortawesome/react-fontawesome": "^0.2.1",
```

Comment:
Yes because I got error that they are not installed, this is exactly the same problem as in #819 
And I pulled the latest changes from the dev (and synced my fork)

Reply:
Hmm, you can remove those two packages I don't think we need it remove from package.json do npm i and commit both package.json and package-lock.json files, maybe last time we missed the part with removing from package.json file